frontend/helpers/issues.ts

```diff
@@ -63,12 +63,19 @@ export const prepareIssue = (issue, jobs = []) => {
 };
 
 export const getStatusIcon = (priority, status) => {
-  const statusIcon = {
-    Icon: STATUSES_ICONS[status] || null,
-    color: STATUSES_COLOURS[status] || STATUSES_COLOURS[priority] || null
-  };
+	const statusIcon = {
+		Icon: STATUSES_ICONS[status] || null,
+		color: (ISSUE_COLORS[status] ? ISSUE_COLORS[status].color : null) ||
```

Comment:
Should this be the same as L77? 
That kind of looks cleaner.

```suggestion
		color: (ISSUE_COLORS[status] || ISSUE_COLORS[priority] || ISSUE_COLORS.NONE).color
```